Lib/idlelib/NEWS.txt

```diff
@@ -3,6 +3,8 @@ Released on 2019-10-20?
 ======================================
 
 
+bpo-35208: Squeezer now counts wrapped lines before newlines.
```

Comment:
This was a bug fix, not an enhancement...
```suggestion
 bpo-35208: Squeezer now properly counts wrapped lines before newlines.
```

Reply:
To me, the first version does not imply enhancement versus bug, but if it does to you, add 'properly' here and in the blurb.